src/components/NewTableComponent/StudentModal.tsx

```diff
@@ -160,19 +256,19 @@ export default function StudentModal({ open, setOpen, id }) {
                                                     <TableCell component="th" scope="row" sx={{ fontWeight: 'bold' }}>
                                                         Name
                                                     </TableCell>
-                                                    <TableCell>{studentData.user.name}</TableCell>
+                                                    <TableCell>{studentData.user ? studentData.user.name : 'N/A'}</TableCell>
                                                 </TableRow>
                                                 <TableRow>
                                                     <TableCell component="th" scope="row" sx={{ fontWeight: 'bold' }}>
                                                         Email
                                                     </TableCell>
-                                                    <TableCell>{studentData.user.email}</TableCell>
+                                                    <TableCell>{studentData.user ? studentData.user.email : 'NA'}</TableCell>
```

Comment:
```suggestion
                                                    <TableCell>{studentData.user ? studentData.user.email : 'N/A'}</TableCell>
```
change it everywhere